DQMServices/Core/python/DQMEDHarvester.py

```diff
@@ -0,0 +1 @@
+from FWCore.ParameterSet.Config import EDProducer as DQMEDHarvester
```

Comment:
I'm afraid I'm pretty much against the idea of injecting additional items in a python module from another module. 
1. It would be easy to forget to add `import DQMServices.Core.DQMEDHarvester` and it happens to work most of the time just because some other module was loaded by the config earlier and that other module had done the needed `import`.
2. It looks like a base framework supported concept but it is not. Therefore people would first look to the framework documentation to see what it is and would not find it there.